tests/nthash.cpp

```diff
@@ -3,172 +3,129 @@
 
 #include <iostream>
 #include <queue>
+#include <stack>
 #include <string>
 
 int
 main()
 {
-  const std::string seq = "ACGTACACTGGACTGAGTCT";
 
   {
-    std::cerr << "Testing single kmer hash values" << std::endl;
-    btllib::NtHash nthash(seq, 3, seq.size());
+    PRINT_TEST_NAME("k-mer rolling")
 
-    const std::vector<uint64_t> hashes = { 10434435546371013747UL,
-                                           16073887395445158014UL,
-                                           8061578976118370557UL };
```

Comment:
I think we would still like to keep hardcoded values in tests (updated with the new canonical hashing), because unless we explicitly decide to change the output of nthash, the values shouldn't change and so hardcoded values can catch bugs where the output changes when it shouldn't.

Reply:
Added in https://github.com/bcgsc/btllib/commit/28c334b0ce8287e34d2e9346332074d30d994b61.